### PR TITLE
NRL-716 role arn var

### PR DESCRIPTION
**Pull request by eesa456**

### CODE REVIEW

File: .github/workflows/pr-env-destroy.yml [hunk 1]
```diff
@@ -84,6 +84,7 @@ jobs:
           terraform -chdir=terraform/infrastructure destroy \
             --var-file=etc/dev.tfvars \
             --var assume_account=${{ steps.get_account_id.outputs.aws_account_id }} \
```
Comment (mattdean3-nhs) [inline .github/workflows/pr-env-destroy.yml]: I think you only need the `role_arn` var now so `-var assume_account=` and `-var assume_role=` can be removed also. However, not needed for the fix the to work, it's just tidy up.

**Review by mattdean3-nhs — APPROVED** [review]
LGTM, nice work :clap: - just one comment about a possible tidy up to remove the old vars.

File: .github/workflows/pr-env-destroy.yml (hunk 1 above)
Comment (eesa456) [reply .github/workflows/pr-env-destroy.yml]: removed

File: .github/workflows/pr-env-destroy.yml [hunk 2]
```diff
@@ -83,8 +83,7 @@ jobs:
         run: |
           terraform -chdir=terraform/infrastructure destroy \
             --var-file=etc/dev.tfvars \
-            --var assume_account=${{ steps.get_account_id.outputs.aws_account_id }} \
-            --var assume_role=terraform \
+            --var assume_role_arn=${{ secrets.DEPLOY_ROLE_ARN }} \
             -auto-approve
```
Comment (Tomdango) [inline .github/workflows/pr-env-destroy.yml:88]: Suggestion - Could also clean up the Terraform workspace

```suggestion
      - name: Cleanup Terraform Workspace
        run: |
          terraform workspace select default
          terraform workspace delete ${{ needs.set-environment-id.outputs.environment_id }}

```

```

**Review by Tomdango — APPROVED** [review]
Approved - but with one possible improvement attached

File: .github/workflows/pr-env-destroy.yml (hunk 2 above)
Comment (eesa456) [reply .github/workflows/pr-env-destroy.yml:88]: added

**Review by Tomdango — APPROVED** [review]
:+1: LGTM

**Pull request merged**